common/history/states/pacific_states.txt

```diff
@@ -116,6 +116,7 @@ STATES = {
 		}
 		
 		add_claim = c:PNG add_homeland = east_papuan
+		add_homeland = melanesian	
```

Comment:
melanesian is to be consolidated into a national identity for each island for game performance reasons so there should be no additional homeland.

Reply:
Amended